includes/blocks/course-theme/class-lesson-actions.php

```diff
@@ -112,16 +112,16 @@ private function render_next_lesson() {
 	 * @return string The take quiz button.
 	 */
 	private function render_take_quiz( string $quiz_permalink, bool $is_disabled ): string {
-		$disabled       = $is_disabled ? 'aria-disabled="true"' : '';
+		$disabled       = $is_disabled ? 'sensei_disable_link' : '';
 		$quiz_permalink = esc_url( $quiz_permalink );
 		$text           = esc_html__( 'Take Quiz', 'sensei-lms' );
 
 		return ( '
-			<form method="GET" action="' . $quiz_permalink . '" class="sensei-course-theme-lesson-actions__take-quiz-form">
-				<button type="submit" data-id="complete-lesson-button" class="sensei-course-theme-lesson-actions__take-quiz sensei-course-theme__button is-primary" ' . $disabled . '>
+			<a href="' . $quiz_permalink . '" class="sensei-course-theme-lesson-actions__take-quiz-form ' . $disabled . '">
```

Comment:
Hum, I think it might break our "required Video" functionality.
We disable the button and prevent sending the form, for Complete Lesson and View Quiz buttons until the user has watched the video.

Reply:
Yeah, turned out it's a bit harder to change this. And there is even a third 'View the lesson quiz' button that's already a link and not working with required blocks. I'll try just changing the form to POST to keep things simple.